environment_mac.yml

```diff
@@ -27,6 +27,7 @@ dependencies:
   - matplotlib>=3.3.3
   - networkx
   - numpy
+  - nodejs
```

Comment:
didnt we add this before?

Reply:
I added `nodejs` to `environment_linux.yml`, but I figured I needed to add the same dependency for `environment_mac.yml`. Let me know if you think this isn't needed.

Reply:
thanks @adisingh50 . Can you add it for Windows, also?